lib/task-data/schemas/types-installos.json

```diff
@@ -5,6 +5,11 @@
             "description": "The type of OS which is used by code internally",
             "enum": [ "esx", "linux", "windows" ]
         },
+        "OsName": {
+            "description": "The name of Linux OS based on LSB Distributor ID (debian, ubuntu)",
+            "type": "string",
+	    "minLength": 1
```

Comment:
indent problem for both line 8 and line 11

Reply:
I cant find any problem with line 8 but i fixed the line 11